### PR TITLE
Add ndm support 

### PR DESCRIPTION
**Pull request by Pradeepkumarbk**
This PR does the following changes :-
-  Add the NDM (node-disk-mansger) dependencies into scope.
-  Add the functionality to list all the Disks under host topology. 
-  Adds the Disk attributes.

Signed-off-by: Pradeepkumarbk <pradeepkumar.bk96@gmail.com>

screenshot_2018-07-30 weave scope

screenshot_2018-07-30 weave scope disk-bfbae361e4322fdc2becc08bf0769ec0 1

### CODE REVIEW

File: render/detailed/summary.go [hunk 1]
```diff
@@ -103,6 +104,7 @@ var primaryAPITopology = map[string]string{
 	report.PersistentVolume:      "pods",
 	report.PersistentVolumeClaim: "pods",
 	report.StorageClass:          "pods",
+	report.Disk:                  "pods",
```
Comment (Akash4927) [inline render/detailed/summary.go]: We're planning to show disks in the `host`. So, I think it should be `host` not `pods`

Comment (qiell) [inline render/detailed/summary.go]: We're planning to show disks in the `host`. So, I think it should be `host` not `pods`

**Review by qiell — CHANGES_REQUESTED** [review]

**Comment by qiell** [comment]
@Pradeepkumarbk Please fix test & linting issue

File: render/detailed/summary.go (hunk 1 above)
Comment (Pradeepkumarbk) [reply render/detailed/summary.go]: @Akash4927 Changed as requested.

**Comment by satyamz** [comment]
Hi @Pradeepkumarbk , 
Thanks for the PR. Changes looks good to me. However, it is lacking the logic to connect disk with host. Lets have that.

**Review by qiell — APPROVED** [review]
lgtm

**Review by satyamz — APPROVED** [review]

**Pull request merged**